wiki/en/Unregistered-Servers.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: wiki
+title: "Running an Unregistered Server"
+lang: "en"
+permalink: "/wiki/Unregistered-Servers"
+---
+
+{% include breadcrumb.html root="More" branch1="Server Administration" branch1-url="Running-a-Server" %}
+
+# Running an Unregistered Server
+
+It is highly recommended to test your Server by registering it on one of the built-in Directories **first** so as to narrow down any subsequent problems in unregistered mode.
```

Comment:
```suggestion
It is highly recommended to test your Server by registering it on one of the built-in Directories **first**. This will help you tackle general problems before you try unregistered mode.
```